apps/ewallet/lib/ewallet/web/v1/serializers/websocket_response_serializer.ex

```diff
@@ -51,9 +51,22 @@ defmodule EWallet.Web.V1.WebsocketResponseSerializer do
   def decode!(message, _opts) do
     decoded = Poison.decode!(message)
 
-    decoded
-    |> Map.put("payload", decoded["data"])
-    |> Message.from_map!()
+    case decoded do
+      %{} = decoded ->
+        decoded
+        |> Map.put("payload", decoded["data"])
+        |> Message.from_map!()
+
+      [join_ref, ref, topic, event, payload | _] ->
```

Comment:
We can remove this now, right? We're not using any pre-made Phoenix stuff anymore.